test/math_test.py

```diff
@@ -105,6 +105,11 @@ def testSequence(self):
         self.assertEqual(v.x, 9.1)
         self.assertEqual(v.y, 11.12)
 
+        v_copy0 = Vector2(2004.0, 2022.0)
+        v_copy1 = v_copy0.copy()
+        self.assertEqual(v_copy0.x, v_copy1.x)
+        self.assertEqual(v_copy0.y, v_copy1.y)
```

Comment:
perhaps these tests must be in their own test function, something like `test_copy`?